### PR TITLE
Fixes #22097

### LINKED ISSUE #22097: Long tag names makes dropdown edit button to be not aligned properly.
### Steps to reproduce

1. Create ~8 tags with small names.
2. Create one tag with 64 characters starting with a character which alphabetically would be ordered before one of them at least.
3. Refresh the page.
4. Open tags dropdown 

### Expected behaviour
Edit buttons remain aligned.

### Actual behaviour
Buttons overlap and lose alignment.

screen shot 2016-02-03 at 12 18 56

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
Master branch
{"installed":true,"maintenance":false,"version":"9.0.0.8","versionstring":"9.0 pre alpha","edition":"Enterprise"}

**Updated from an older ownCloud or fresh install:**
Fresh

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No.

**Are you using encryption:** 
No

**Logs**

```
```

### Client configuration

**browser**
Chrome, Edge, Firefox

### PR DESCRIPTION
**Pull request by ErikPel**

### CODE REVIEW

File: core/css/systemtags.css [hunk 1]
```diff
@@ -76,4 +76,10 @@
 .systemtags-select2-container .select2-choices .select2-search-field input {
 	line-height: 20px;
 }
-
+.systemtags-item > .label
```
Comment (PVince81) [inline core/css/systemtags.css]: Can you make it more specific, at least add ".systemtags-select2-dropdown" because it's only for the dropdown

**Comment by PVince81** [comment]
Nice one!

@ErikPel Thanks a lot for your contribution!
Contributions to the core repo require a signed contributors agreement http://owncloud.org/about/contributor-agreement/

Alternatively you can add a comment here where you state that this contribution is MIT licensed.

Some more details about out pull request workflow can be found here: http://owncloud.org/code-reviews-on-github/

**Comment by ErikPel** [comment]
Is the css selector better now? @PVince81 

I have already signed the contributors agreement.
Check the comments here:
https://github.com/owncloud/core/pull/21599#issuecomment-171309124

**Comment by PVince81** [comment]
That's great, I don't have the list so wasn't sure. Sorry :smile: 

The code change looks better, I'll test it later

**Comment by rullzer** [comment]
@ErikPel very nice thanks!

Now that you are a collaborator you should also be able to create branches in the ownCloud repository. If you then create a PR it is easier for reviewers to get and test your code. And all our tests scripts can do their job properly. 

It is fine for now. But it makes reviewing your future PR's a bit easier for us.

**Comment by rullzer** [comment]
O and please squash your commits :)

**Comment by ErikPel** [comment]
@rullzer Didn't know I could do that. Have to do it next time then :)

Squashed my commits

**Comment by SergioBertolinSG** [comment]
Nice, but now it has a problem with the :heavy_check_mark:'s on the left.

screen shot 2016-02-03 at 15 50 00

**Comment by ErikPel** [comment]
Which browser is that so I can reproduce and fix?
@SergioBertolinSG

**Comment by SergioBertolinSG** [comment]
I've seen it on firefox and chrome.

**Comment by ErikPel** [comment]
@SergioBertolinSG is that follow up problem happening on chrome too?

I have no problem with mine

**Comment by ErikPel** [comment]
Does it work now @SergioBertolinSG ?

**Comment by SergioBertolinSG** [comment]
Yes :+1:

**Comment by ErikPel** [comment]
@SergioBertolinSG Nice! :)

**Comment by MorrisJobke** [comment]
Tested and works :+1: 

CSS only change -> no CI needed

**Pull request merged**